api-server/api/openapi.yaml

```diff
@@ -135,6 +135,40 @@ paths:
         "204":
           description: success
       summary: Register interest for a sort key
+  /events/feed:
```

Comment:
so, the firehose endpoint in js-ceramic is at `/api/v0/feed/aggregation/documents`.  If we want to use a consistent convention then this one should probably be `/feed/events` instead of `events/feed`?  I'm assuming the `/api/v0` is already handled somehow elsewhere?

Reply:
> I stuck to the "spec" in the doc, but I originally added a new type to wrap the body e.g. something like {eventids: [string] } where we could add new fields

What if instead, the objects that came back in responses from the feed were like:
```
{
  events: [ {id: "<eventId1>" }, { id: "<eventId2>"}, ... ]
  resumeToken: "XXXXX",
}```

That way if in the future we add an arg to the endpoint to instruct it to start including event payload data too, we could add a `data` field alongside the `id` field in the inner `event` objects?